packages/core/helper-plugin/lib/src/components/ReactSelect/index.js

```diff
@@ -0,0 +1,3 @@
+import ReactSelect from './ReactSelect';
```

Comment:
Why don't you just use this file to code the component instead of importing and exporting?

Reply:
I really like the style of having an `index.js` and `[Component].js` file as we do in some parts of the admin, because it makes it really easy to navigate to that file using the editor ...